src/CSharpLanguageServer/State.fs

```diff
@@ -396,6 +396,24 @@ type ServerRequestScope (requestId: int, state: ServerState, emitServerEvent, lo
         return aggregatedLspLocations
     }
 
+
+    member scope.ResolveTypeSymbolLocations
+            (project: Microsoft.CodeAnalysis.Project)
+            (symbols: Microsoft.CodeAnalysis.ITypeSymbol list) = async {
+        let! ct = Async.CancellationToken
+        let! compilation = project.GetCompilationAsync(ct) |> Async.AwaitTask
+
+        let mutable aggregatedLspLocations = []
+
```

Comment:
There's probably a way to do this without code duplication, but I'm not that great at F#. Maybe a type discriminated union would work here? I was surprised that `ITypeSymbol` does not inherit from `ISymbol`